src/main/steps/common/common.content.ts

```diff
@@ -175,6 +175,7 @@ export const generatePageContent = ({
   const partner = getPartnerContent(commonTranslations, selectedGender, isDivorce);
   const contactEmail = isDivorce ? 'contactdivorce@justice.gov.uk' : 'civilpartnership.case@justice.gov.uk';
   const isJointApplication = formState?.applicationType === ApplicationType.JOINT_APPLICATION;
+  const hasAppliedForHWF = formState?.applicant1HelpPayingNeeded === YesOrNo.YES;
```

Comment:
as this is for 1 specific page, would it not be best to have it in the content file for that page?